includes/Checker/Abstract_Check_Runner.php

```diff
@@ -58,6 +58,14 @@ abstract class Abstract_Check_Runner implements Check_Runner {
 	 */
 	protected $plugin_basename;
 
+	/**
+	 * An instance of the Check_Repository.
+	 *
+	 * @since n.e.x.t
+	 * @var Check_Repository
+	 */
+	protected $check_repository;
```

Comment:
Please keep in mind to use the lowest visibility feasible, i.e. this should be `private`.